.maint/update_zenodo.py

```diff
@@ -152,9 +150,7 @@ def loads_contributors(s):
     zenodo['contributors'] = zen_contributors
 
     print(
-        'Some people made commits, but are missing in .maint/ files: {}.'.format(
-            ', '.join(set(miss_creators).intersection(miss_contributors))
-        ),
+        f'Some people made commits, but are missing in .maint/ files: {", ".join(set(miss_creators).intersection(miss_contributors))}.',
```

Comment:
A bit shorter to use unpacking and the `&` operator.

```suggestion
        f'Some people made commits, but are missing in .maint/ files: {", ".join({*miss_creators} & {*miss_contributors})}.',
```

TBH I think it should probably be something like:

```py
missing = {*miss_creators} & {*miss_contributors}
if missing:
    print(f'Some people made commits, but are missing in .maint/ files: {", ".join(sorted(missing))}.', file=sys.stderr)
```